packages/base-controller/src/ControllerMessenger.ts

```diff
@@ -129,6 +129,16 @@ export class ControllerMessenger<
 
   readonly #events = new Map<Event['type'], EventSubscriptionMap<Event>>();
 
+  /**
+   * A map of functions for getting the initial event payload.
+   *
+   * Used only for events that represent state changes.
+   */
+  readonly #getEventPayload = new Map<
```

Comment:
The name of this implies that this is a function, not a map of event types to functions.

It's not much better but perhaps `#eventPayloadGetters` makes more sense? Or `#initialEventPayloadGetters`, to more closely match how this is used in practice?

Reply:
Good point, I'll go with `initialEventPayloadGetters`